classes/class06/hw6/public/javascripts/main.js

```diff
@@ -0,0 +1,79 @@
+var $addForm = $("#add-form");
+var $logForm = $("#log-form");
+var $list = $(".twot-list");
+var $logger = $('.logButton');
+var $highlighter = $('.userButton');
+var $del = $('.trashButton');
+
+var Success = function(data, status) {
+  console.log('Successful Form Submission');
+};
+
+var Error = function(data, status) {
+  console.log("status", status);
+  console.log("error", data);
+};
+
+var adder = function(twot) {
+  $list.prepend('<div class="twot '+ twot.author +'"><div class="twot-m">'+ twot.message + '</div><div class="twot-a">' + twot.author + '</div><button type="button" class="trashButton" id="mine">X</button></div>');
```

Comment:
If you just add the element, it doesn't have any of the necessary events bound (the delete action for the delete button in this case). You'd need to manually bind the events here for the element to work without refreshing the page.